pkg/reconciler/instances/warden/warden.go

```diff
@@ -0,0 +1,24 @@
+package warden
+
+import (
+	"github.com/kyma-incubator/reconciler/pkg/logger"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/service"
+)
+
+const ReconcilerName = "warden"
+
+//nolint:gochecknoinits //usage of init() is intended to register reconciler-instances in centralized registry
+func init() {
+	log := logger.NewLogger(false)
+
+	log.Debugf("Initializing component reconciler '%s'", ReconcilerName)
+	reconciler, err := service.NewComponentReconciler(ReconcilerName)
+	if err != nil {
+		log.Fatalf("Could not create '%s' component reconciler: %s", ReconcilerName, err)
+	}
+
+	reconciler.
+		WithPreReconcileAction(&CleanupWardenAdmissionCertColumeMounts{
+			name: "cleanup-warden-admissio-cert-volume-mounts-action",
```

Comment:
What is this `name` used for?

Reply:
yes. Now  it is used in log that is the first line in the `Run` function.